Resources/config/renderer.xml

```diff
@@ -6,7 +6,12 @@
 >
     <services>
         <service id="ivory_ck_editor.renderer" class="Ivory\CKEditorBundle\Renderer\CKEditorRenderer">
-            <argument type="service" id="service_container" />
+            <argument type="service" id="ivory_ck_editor.renderer.json_builder" />
+            <argument type="service" id="router" />
+            <argument type="service" id="request_stack" />
+            <argument type="service" id="assets.packages" on-invalid="null" />
+            <argument type="service" id="templating" on-invalid="null" />
+            <argument></argument>
```

Comment:
it seems is defaulting to null do you need to send this argument here?

Reply:
It is necessary for replacing it later on in the extension, otherwise you will receive an error when attempting to replace an argument with an index out of range.